src/main/java/core/basesyntax/TriangleArea.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface TriangleArea {
+    default double getTriangleArea(double firstLeg, double secondLeg) {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. Also, avoid using default methods in interfaces to set behavior for classes. Consider refactoring this by defining the method signature without an implementation and implementing the behavior in the appropriate classes.